### DISCUSSION TITLE
Application "unknown"

### DISCUSSION THREAD

**Discussion by romanix** [phantom/docs#196] [Help & Support] [answered]
Our application appears as "unknown" in Phantom (and in Solflare) wallet.

What do we need to change or add?

![Screenshot_20240312-010951~2.png](https://github.com/phantom/docs/assets/110880/600bd77e-5d9e-4ba3-a5ce-9d33ba312ee0)

**Comment by adamdelphantom** [comment, marked as answer]
Hello @romanix,

Here are our best practices for displaying app names in Phantom.

https://docs.phantom.app/best-practices/displaying-your-app

**Reply by romanix** [reply]
Great. Thanks. 

Next time RTFM first, I guess :-)

**Reply by alexandr-kazakov** [reply]
@adamdelphantom Hi,

Thank you for you reply, we tried it and it doesn't help us. Some of the largest projects has the same issue, see example below:

Try to go to https://solend.fi/dashboard from Android phone in Chrome browser and connect via Phantom wallet after that you'll see the same thing as in my screenshot.

![Screenshot_20240316_232824_Phantom](https://github.com/phantom/docs/assets/20513262/4fa22618-c6af-4b31-b857-2af198789be8)

I tried the different combinations of titles and icons with logo on our web app pages and it doesn't help, it still has Unknown title without project logo.

**Reply by adamdelphantom** [reply]
This might be a limitation of deeplinking in. I'll investigate further.

Do you see the same issue when using the phantom mobile dapp browser?

**Reply by alexandr-kazakov** [reply]
> Do you see the same issue when using the phantom mobile dapp browser?

No. If I go to a app directly from phantom browser I don't see such issue, in this case I see another connect window and it works fine, see screenshot below:

![Screenshot_20240324_132622_Phantom](https://github.com/phantom/docs/assets/20513262/a072901b-3746-424b-9ec1-1e48d983bd79)

This issue appears in the next flow in Android mobile phones:

1). Go to a app from Chrome mobile browser in Android mobile device.
2). Click Connect via Phantom button. In wallet-adapter(https://github.com/anza-xyz/wallet-adapter) case will be automatically opened  Phantom mobile Android app and window with Connect options with Unknown text.

**Reply by adamdelphantom** [reply]
Thanks for the further info!

This is a bug on our side. We will continue to investigate and fix. Thanks for reporting!

**Comment by romanix** [comment]
Yes, I got the notification as well. This bug has cost you time and hence has cost me money.
